thanosql/resources/_query.py

```diff
@@ -26,7 +27,7 @@ class QueryLog(BaseModel):
     destination_schema: Optional[str]
     error_result: Optional[str]
     created_at: Optional[datetime]
-    records: Optional[list] = None
+    records: Optional[Record] = None
```

Comment:
```suggestion
    records: Optional[Records] = None
```
Optional[List(Record)] 이어야만 할 것 같아요. Record -> Records로 바꾸는게 좋겠습니다.